heartbeat/monitors/util.go

```diff
@@ -342,3 +444,42 @@ func filterIPs(ips []net.IP, filt func(net.IP) bool) []net.IP {
 	}
 	return out
 }
+
+func MakeJobSetting(name string) JobSettings {
+	return JobSettings{Name: name}
+}
+
+func (s JobSettings) WithFields(m common.MapStr) JobSettings {
+	s.AddFields(m)
+	return s
+}
+
+func (s *JobSettings) AddFields(m common.MapStr) { addFields(&s.Fields, m) }
+
+func MakeHostJobSettings(name, host string, ip IPSettings) HostJobSettings {
+	return HostJobSettings{Name: name, Host: host, IP: ip}
+}
+
+func (s HostJobSettings) WithFields(m common.MapStr) HostJobSettings {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method HostJobSettings.WithFields should have comment or be unexported